addons/ai/fnc_taskAttack.sqf

```diff
@@ -31,4 +31,6 @@ params ["_group","_position", ["_radius",0]];
 _group = _group call CBA_fnc_getGroup;
 if !(local _group) exitWith {}; // Don't create waypoints on each machine
 
+[_group] call CBA_fnc_clearWaypoints; // Clear existing waypoints

```

Comment:
A comment should explain why a piece of code does what it does. Ie a very brief summary of what you wrote in the PR.